x-pack/plugins/synthetics/server/routes/monitor_cruds/add_monitor_project.ts

```diff
@@ -62,23 +70,23 @@ export const addSyntheticsProjectMonitorRoute: SyntheticsStreamingRouteFactory =
         server,
         syntheticsMonitorClient,
         request,
-        subject,
       });
 
       await pushMonitorFormatter.configureAllProjectMonitors();
 
-      subject?.next({
+      return {
         createdMonitors: pushMonitorFormatter.createdMonitors,
         updatedMonitors: pushMonitorFormatter.updatedMonitors,
```

Comment:
will POST also handle updates? i guess if we want a true REST, i imagine PUT handling updates.

Reply:
I believe Andrew sees this as an operation happening on the **_project_**, rather than the monitors themselves being individual entities.